api/http/server.go

```diff
@@ -349,6 +352,17 @@ func (s *Server) HandlePostFiles(w http.ResponseWriter, r *http.Request) {
 	log.Debug("handle.post.files", "ruid", ruid)
 	postFilesCount.Inc(1)
 
+	tagUID := sctx.GetTag(r.Context())
+	tag, err := s.api.Tags.Get(tagUID)
+	if err != nil {
+		log.Error("handle post raw got an error retrieving tag", "tagUID", tagUID, "err", err)
+	}
+
+	ctx := r.Context()
+
+	_, sp := spancontext.StartSpan(tag.Context(), "http.post")
```

Comment:
I think that returned context should be used after this function call `ctx, sp := spancontext.StartSpan(tag.Context(), "http.post")`, and in that case, the ctx assignment above is not needed.